### PR TITLE
OHM-429: Mentioning about keystore is helpful to the devs

### PR DESCRIPTION
**Pull request by kaweesi**
* consider forexample attempting to resolve: https://jembiprojects.jira.com/browse/OHM-382

### CODE REVIEW

File: docs/user-guide/certificates.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-Certificates
+Certificates & Keystore
 ============
 
-The OpenHIM has a built in capability to manage TLS certificates. You can upload a certificate and key that you have bought from a certificate authority such as Thwate or you can even generate your own self signed certificate to use in your private OpenHIM implementation. Both mechanisms are secure, however we suggest you purchase a certificate from a trusted certificate authority so save you some pain with self signed certificates.
+The OpenHIM has a built in capability to manage TLS certificates and keys through its keystore. You can upload a certificate and key that you have bought from a certificate authority such as Thwate or you can even generate your own self signed certificate to use in your private OpenHIM implementation. Both mechanisms are secure, however we suggest you purchase a certificate from a trusted certificate authority so save you some pain with self signed certificates.
```
Comment (napiergit) [inline docs/user-guide/certificates.md]: I know this was not you but there is a typo: 'so save you some pain' should be: 'to save you some pain'.

**Comment by napiergit** [comment]
Hi @kaweesi this looks good, just a minor comment.

File: docs/user-guide/certificates.md (hunk 1 above)
Comment (kaweesi) [inline docs/user-guide/certificates.md]: change made at: 67bae94a73a9e63fed11864ac8b72fa7342e03db

**Comment by napiergit** [comment]
LGTM :+1:

**Pull request merged**